src/engine/renderer/glsl_source/vertexLighting_DBS_entity_fp.glsl

```diff
@@ -103,14 +97,6 @@ void	main()
 	// compute the material term
 	vec4 material = texture2D(u_MaterialMap, texCoords);
 
-#if defined(USE_REFLECTIVE_SPECULAR) && !defined(USE_PHYSICAL_MAPPING)
```

Comment:
computeLight is called in several places, but this code only exists here. Is that an intentional change?

Reply:
Yes, in my agenda the next factorisation step is to reduce the number of places calling computeLight. Basically the whole file containing this code (copy pasting 90% of another file not containing this code) exists because of this code not being in computeLight.